score-server/src/main/java/bio/overture/score/server/controller/BaseController.java

```diff
@@ -0,0 +1,22 @@
+package bio.overture.score.server.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RestController
+public class BaseController {
+
+    @Autowired
+    Environment environment;
+
+  @RequestMapping(method = RequestMethod.GET, value = "/profile")
+  public List<String> getProfile(@RequestHeader(HttpHeaders.AUTHORIZATION) final String accessToken) {
+      return Arrays.asList(environment.getActiveProfiles());
```

Comment:
Two thoughts here:
1. We don't want to reveal to the public all our runtime profiles
2. We should have separation between our runtime profile names and the storage type that we are telling the client to operate in.

So can we create an enum in score-core that can be shared between client and server. This can list the storage types that score supports (S3, Azure). This method should return only one of these enum values, and the client can then make its switch based on that value, instead of having to look through a list and do string checks.